CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - Added `safeScrollToRow(at:at:animated:)` method to safely scroll UITableView to the given IndexPath. [#445](https://github.com/SwifterSwift/SwifterSwift/pull/445) by [setoelkahfi](https://github.com/setoelkahfi).
 - **Optional**:
   - Added `isNilOrEmpty` property to check whether an optional is nil or empty collection.
+- Added SpriteKitExtensions to docs. [#493](https://github.com/SwifterSwift/SwifterSwift/pull/493) by [@oliviabrown9](https://github.com/oliviabrown9)
```

Comment:
@oliviabrown9 Thanks for trying to conform to our guidelines as close as possible 👍 We actually don't require CHANGELOG entries for non-code related changes. You'll notice we have the SwifterSwift bot running Danger. If the pull request detects a change in the source files, it [will warn you that you need to add a CHANGELOG entry](https://github.com/SwifterSwift/SwifterSwift/blob/4903b2394ae8f673c83e0cd0f6a1765df2f58a9f/Dangerfile#L16-L18) (if you haven't done so already).